src/status_im/models/transactions.cljs

```diff
@@ -177,12 +178,17 @@
     (:mainnet) "api"
     (:rinkeby) "api-rinkeby"))
 
-(defn- get-transaction-url [chain account]
-  {:pre [(keyword? chain) (string? account)]
-   :post [(string? %)]}
-  (let [network-subdomain (get-api-network-subdomain chain)]
-    (str "https://" network-subdomain ".etherscan.io/api?module=account&action=txlist&address=0x"
-         account "&startblock=0&endblock=99999999&sort=desc&apikey=" etherscan-api-key "&q=json")))
+(defn- get-transaction-url
+  ([chain account] (get-transaction-url chain account false))
+  ([chain account chaos-mode?]
+   {:pre  [(keyword? chain) (string? account)]
+    :post [(string? %)]}
+   (let [network-subdomain (get-api-network-subdomain chain)]
+     (str "https://" network-subdomain
+          (if chaos-mode?
+            "some-broken-url-af435c85ff48c2b9bdefd1295678fc24.io/api?module=account&action=txlist&address=0x"
```

Comment:
you could also use `http://httpstat.us/500` so it's more realistic (this will throw a dns error likely), but minor thing

Reply:
ok

Reply:
fixed